docker-compose.yml

```diff
@@ -37,6 +37,16 @@ services:
       - "127.0.0.1:5000:5000"
     networks:
       - public-net
+
+  identity-functest:
+    build:
+      context: ./functionaltest/docker
+    network_mode: "host"
```

Comment:
It is for the `functest` container to be able to access the same "localhost" as the host machine. Without `network_mode` : "host" the tests cannot navigate to the "localhost" of the host machine. 
Maybe there are other ways to do the same thing, like replace the urls that I hardcoded into the test cases with the public ip address of the host machine, however that's more steps and since travis does not give you the same machine all the time I figured it's better just to map the host mahine's "localhost" to the `functest` container.

Reply:
It should connect to the frontend using hostname `identity-frontend`.